contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -1375,10 +1379,10 @@ execute_batch(PLtsql_execstate *estate, char *batch, InlineCodeBlockArgs *args,
 			 */
 
 			/* Safety check */
-			if (fcinfo->nargs > FUNC_MAX_ARGS)
+			if (fcinfo->nargs > list_length(params) + 2)
```

Comment:
Why we are not using PREPARE_STMT_MAX_ARGS here?

Reply:
On line 2250 we have already added that check for max number of arguments. This is a sanity check that fcinfo->nargs which is allocated with nargs + 2 size doesnt go over the limit of number of bind params